src/main/java/org/apache/commons/lang3/ObjectUtils.java

```diff
@@ -226,6 +227,62 @@ public static boolean anyNotNull(final Object... values) {
         return firstNonNull(values) != null;
     }
 
+    /**
+     * <p>
+     * Invokes the given {@code consumer's} {@link Consumer#accept(Object)} with the first {@code non-null} value from
```

Comment:
IMO, you're not "invoking", you're "accepting", so the Javadoc should say "Accepts...". Actually, the method is misnamed since the underlying method is `Consumer#accept()`, this method should be use "accept", not "apply".

Reply:
I have changed the method name to 'accept...' and updated the Javadoc comment accordingly.